supportedBackends/htcondor/src/main/scala/cromwell/backend/impl/htcondor/CondorJobExecutionActor.scala

```diff
@@ -0,0 +1,186 @@
+package cromwell.backend.impl.htcondor
+
+import java.nio.file.attribute.PosixFilePermission
+import java.nio.file.{Files, Path, FileSystems}
+import java.util.regex.Pattern
+
+import akka.actor.Props
+import cromwell.backend.BackendJobExecutionActor.{SucceededResponse, FailedNonRetryableResponse, BackendJobExecutionResponse}
+import cromwell.backend._
+import wdl4s.util.TryUtil
+
+import scala.annotation.tailrec
+import scala.concurrent.Future
+import scala.sys.process.ProcessLogger
+import scala.util.{Failure, Success, Try}
+
+object CondorJobExecutionActor {
+
+  val fileSystems = List(FileSystems.getDefault)
+
+  def props(jobDescriptor: BackendJobDescriptor, configurationDescriptor: BackendConfigurationDescriptor): Props =
+    Props(new CondorJobExecutionActor(jobDescriptor, configurationDescriptor))
+
+}
+
+class CondorJobExecutionActor(override val jobDescriptor: BackendJobDescriptor,
+                              override val configurationDescriptor: BackendConfigurationDescriptor) extends BackendJobExecutionActor with SharedFileSystem {
+
+
+  import CondorJobExecutionActor._
+  import cromwell.core.PathFactory._
+  import better.files._
+
+  lazy val cmds = new HtCondorCommands {}
+  lazy val extProcess = new HtCondorProcess {}
+  lazy val parser = new HtCondorClassAdParser {}
```

Comment:
Why not make those (`HtCondorCommands`...) classes instead of traits if you want to instantiate them directly ?